packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -338,6 +343,37 @@ async def get_transaction_receipt(
             )
         return response
 
+    @staticmethod
+    def _get_transaction_digest(
+        api: LedgerApi, message: LedgerApiMessage
+    ) -> Optional[str]:
+        """Send a signed transaction and get the transaction digest."""
+        if message.rpc_config and message.rpc_config.body.get(
+            "use_flashbots", DEFAULT_USE_FLASHBOTS
+        ):
+            target_blocks = message.rpc_config.body.get("target_block_numbers", None)
+            if target_blocks is None:
+                current_block_number = int(api.api.eth.get_block_number())
+                target_blocks = [
+                    *range(
+                        current_block_number,
+                        current_block_number + DEFAULT_NEXT_TARGET_BLOCKS,
+                    )
+                ]
+            target_blocks = cast(List[int], target_blocks)
+
+            # at the moment we do not support multiple transactions, though flashbots can support this
+            raw_transaction = str(message.signed_transaction.body["raw_transaction"])
+            tx_hashes = api.bundle_and_send([raw_transaction], target_blocks)
+            if tx_hashes is not None:
+                return tx_hashes[0]
```

Comment:
What happens to the other hashes?

Reply:
Ok, so this is because we only send a single raw transaction. Is there a way we can generalise this too?